tests/tester.js

```diff
@@ -246,12 +245,14 @@ function runTests (name, runnerArgs, cb) {
 }
 
 function runAll () {
-  require('./tester.js')
+  console.log('[INFO] Generic test cmd not used. See package.json for more specific test run cmds.')
```

Comment:
Same here, I think it should be fine to just delete this, as the tests don't need to maintain backwards compatibility or anything. In `package.json`, the `test` script is calling this here, not sure what it should be changed to, maybe `testAPI`?

Reply:
Wasn't sure in both cases, but I am feeling a bit thankful for both comments now. 😂 Will delete both.

Reply:
Have also deleted. I have now placed the info message directly into the npm test command, think that's most practically useful especially for newcomers to have some guidance when they start with `npm run test`. Wasn't any good reasoning (as well as use case) to include some specific test run command in here.